File: Java/app/src/main/java/org/mobileProgramming/maintermproject/model/ChatModel.java
package org.mobileProgramming.maintermproject.model;

import java.util.HashMap;
import java.util.Map;

public class ChatModel {
    public Map<String, Boolean> users = new HashMap<>(); // 채팅방 유저들
    public Map<String, Comment> comments = new HashMap<>(); //채팅방의 내용
    public static class Comment {
        public String uid;
        public String message;

    }
}

File: Kotlin/Android_PetCommunity/app/src/main/java/com/sehun/android_pet_community/Model/ChatModel.kt
package com.sehun.android_pet_community.Model

data class ChatModel (
    var usersList : List<String> = listOf(),
    var users :MutableMap<String, Boolean>? = mutableMapOf(),
    var comments : List<Map<String, String>> = listOf() )

{
    data class Comment(
        var uid : String = "",
        var message : String = "",
    )
}

